tests/core/gui/test_main_window.py

```diff
@@ -708,3 +714,98 @@ async def test_link_file_unknown_org(
     assert accounts_w
 
     assert isinstance(accounts_w, LoginPasswordInputWidget)
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+async def test_join_organization_org_link_in_clipboard(
+    aqtbot,
+    running_backend,
+    backend,
+    autoclose_dialog,
+    gui,
+    catch_text_input_widget,
+    invitation_organization_link,
+    qt_thread_gateway,
+):
+    def _copy_to_clipboard():
+        desktop.copy_to_clipboard(invitation_organization_link)
+
+    await qt_thread_gateway.send_action(_copy_to_clipboard)
+
+    await aqtbot.key_click(gui, "o", QtCore.Qt.ControlModifier, 200)
+    text_input_w = await catch_text_input_widget()
+    assert text_input_w
+
+    assert text_input_w.line_edit_text.text() == invitation_organization_link
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+async def test_join_organization_device_link_in_clipboard(
```

Comment:
It seems that only the invitation link is different from the previous test `test_join_organization_org_link_in_clipboard`. Maybe those two tests can be merged in one parameterized test.